Hi again,

do you have any idea to set up the tests correctly? I update the `key` and `id` fields in `ec2.conf` but I still got errors as below

```
# EXPENSIVE_TESTS=True python tests/runtests.py  -n integration.cloud.providers.test_ec2
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 * Python Version: 2.7.12 (default, Dec 4 2017, 14:50:18) [GCC 5.4.0 20160609]
 * Transplanting configuration files to '/tmp/salt-tests-tmpdir/config'
 * Current Directory: /root/salt
 * Test suite is running under PID 15716
 * Logging tests on /tmp/salt-runtests.log
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 * Max open files settings is too low (soft: 1024, hard: 1048576) for running the tests
 * Trying to raise the limits to soft: 3072, hard: 1048576
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 * Setting up Salt daemons to execute tests
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 * Starting salt-master ... STARTED!                                                                                                                                                                                                                                                      
 * Starting salt-minion ... STARTED!                                                                                                                                                                                                                                                      
 * Starting sub salt-minion ... STARTED!                                                                                                                                                                                                                                                  
 * Starting syndic salt-master ... STARTED!                                                                                                                                                                                                                                               
 * Starting salt-syndic ... STARTED!                                                                                                                                                                                                                                                      
==========================================================================================================================================================================================================================================================================================
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Starting integration.cloud.providers.test_ec2 Tests
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/tmp/salt-tests-tmpdir/scripts/salt-cloud", line 2, in <module>
    from salt.scripts import salt_cloud
ImportError: No module named salt.scripts
sTraceback (most recent call last):
  File "/tmp/salt-tests-tmpdir/scripts/salt-cloud", line 2, in <module>
    from salt.scripts import salt_cloud
ImportError: No module named salt.scripts
sTraceback (most recent call last):
  File "/tmp/salt-tests-tmpdir/scripts/salt-cloud", line 2, in <module>
    from salt.scripts import salt_cloud
ImportError: No module named salt.scripts
ssTraceback (most recent call last):
  File "/tmp/salt-tests-tmpdir/scripts/salt-cloud", line 2, in <module>
    from salt.scripts import salt_cloud
ImportError: No module named salt.scripts
ss
----------------------------------------------------------------------
Ran 6 tests in 0.836s

OK (skipped=6)

=================================================================================================================================  Overall Tests Report  =================================================================================================================================
*** integration.cloud.providers.test_ec2 Tests  ******************************************************************************************************************************************************************************************************************************************
 --------  Skipped Tests  ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
   -> integration.cloud.providers.test_ec2.EC2Test.test_instance          ->  Configuration file for ec2 was not found. Check ec2.conf files in tests/integration/files/conf/cloud.*.d/ to run these tests.
   -> integration.cloud.providers.test_ec2.EC2Test.test_instance_rename   ->  Configuration file for ec2 was not found. Check ec2.conf files in tests/integration/files/conf/cloud.*.d/ to run these tests.
   -> integration.cloud.providers.test_ec2.EC2Test.test_win2012r2_psexec  ->  Configuration file for ec2 was not found. Check ec2.conf files in tests/integration/files/conf/cloud.*.d/ to run these tests.
   -> integration.cloud.providers.test_ec2.EC2Test.test_win2012r2_winrm   ->  Skip when winrm dependencies are missing
   -> integration.cloud.providers.test_ec2.EC2Test.test_win2016_psexec    ->  Configuration file for ec2 was not found. Check ec2.conf files in tests/integration/files/conf/cloud.*.d/ to run these tests.
   -> integration.cloud.providers.test_ec2.EC2Test.test_win2016_winrm     ->  Skip when winrm dependencies are missing
 -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
==========================================================================================================================================================================================================================================================================================
OK (total=6, skipped=6, passed=0, failures=0, errors=0) 
=================================================================================================================================  Overall Tests Report  =================================================================================================================================
```

Thanks alot

--- a/salt/utils/aws.py
+++ b/salt/utils/aws.py
@@ -137,6 +137,7 @@ def creds(provider):
 
     return ret_credentials
 
+
 def sig2(method, endpoint, params, provider, aws_api_version):
     '''
     Sign a query against AWS services using Signature Version 2 Signing

